docs/pom.xml

```diff
@@ -45,6 +45,8 @@
                         <quarkus-version>${project.version}</quarkus-version>
                         <surefire-version>${version.surefire.plugin}</surefire-version>
                         <graalvm-version>${graal-sdk.version-for-documentation}</graalvm-version>
+                        <jdk-version>${jdk-version}</jdk-version>
+                        <maven-version>${maven-version}</maven-version>
```

Comment:
How does it work? From where does `${maven-version}` come from?

Reply:
My bad, I suggested this change earlier, but wasn't super clear. I guess it should be the Maven equivalent to the new attributes in https://github.com/quarkusio/quarkusio.github.io/pull/247/files#diff-2e69486449bbf7e685101a3ddcbf0d5b:
i.e:
```xml
<jdk-version>JDK 8 or 11</jdk-version>
<maven-version>Apache Maven 3.5.4+</maven-version>
```

Reply:
How is it that some attributes in `pom.xml` can use the `${}` syntax without explicitly defining the value (i.e., ` <quarkus-base-url>${quarkus-base-url}</quarkus-base-url>`)?

Reply:
These are defined as Maven properties:
https://github.com/quarkusio/quarkus/blob/f226fccfb941a1e6ce0bd76d809274996f4ba75b/docs/pom.xml#L21
https://github.com/quarkusio/quarkus/blob/f226fccfb941a1e6ce0bd76d809274996f4ba75b/build-parent/pom.xml#L34

Reply:
Thank you for clarifying and sorry for the confusion. I've gone ahead and pushed up the changes to `pom.xml`. Let me know if you would like me to squash commits.